routes/web.php

```diff
@@ -1,53 +1,67 @@
 <?php
+namespace App\Http\Controllers;
 
-Route::get('/', 'HomeController@index')->name('home');
-Route::get('/page/{page}', 'HomeController@index')->name('home.page');
+use Illuminate\Support\Facades\Route;
 
-Route::get('faq', 'FaqController@view')->name('faq');
-Route::get('faq/list', 'FaqController@listFaqs')->name('faq.list');
-Route::match(['get', 'post'], 'faq/add', 'FaqController@edit')->name('faq.add');
-Route::match(['get', 'post'], 'faq/edit/{id}', 'FaqController@edit')->name('faq.edit');
-Route::match(['get', 'post'], 'faq/delete/{id}', 'FaqController@delete')->name('faq.delete');
+/*
+|--------------------------------------------------------------------------
+| Web Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your application. These
+| routes are loaded by the RouteServiceProvider and all of them will
+| be assigned to the "web" middleware group. Make something great!
+|
+*/
+
+Route::get('/', [HomeController::class, 'index'])->name('home');
+Route::get('/page/{page}', [HomeController::class, 'index'])->name('home.page');
+
+Route::get('faq', [FaqController::class, 'view'])->name('faq');
+Route::get('faq/list', [FaqController::class, 'listFaqs'])->name('faq.list');
+Route::match(['get', 'post'], 'faq/add',  [FaqController::class, 'edit'])->name('faq.add');
+Route::match(['get', 'post'], 'faq/edit/{id}',  [FaqController::class, 'edit'])->name('faq.edit');
+Route::match(['get', 'post'], 'faq/delete/{id}',  [FaqController::class, 'delete'])->name('faq.delete');
 
 // Login and log out
-Route::get('login/{token}', 'LoginController@loginByToken');
-Route::match(['get', 'post'], 'login', 'LoginController@login')->name('login');
-Route::post('logout', 'LoginController@logout')->name('logout');
+Route::get('login/{token}',  [LoginController::class, 'loginByToken']);
```

Comment:
Maybe this route could also have a name for consistencyy

Reply:
The reason I didn't add a name to this one in the past, is because with a name you allow to redirect/link to it through code. So in a way I tried to enforce here that you can only do it by manually putting in the URL in the browser